### DISCUSSION TITLE
Help starting the server

### DISCUSSION THREAD

**Discussion by moonrise2375** [gekigek99/minecraft-vanilla-server-hibernation#107] [Help required] [answered]
I can't figure out how to start the server, everything else is working fine. Right now I have to run msh and manually type "msh start" every time I want to start the server. As I understand I need to open the port 25555 in my router, the problem is that I can't access the settings for my router, meaning I can't open any port. Is there a way to start/awake the server without opening ports?

**Comment by gekigek99** [comment, marked as answer]
can you describe a little bit whats your situation:

- You have a machine on which the msh is running (with the minecraft server), and **on the same LAN** you have your pc with the minecraft client right?
- If you type `msh start`, the minecraft server is launched? (can you see the mineraft server logs?)
- if you launch the minecraft server without using msh (the standard way), are you able to connect to the server?
- Do you have a firewall on the server machine?

(opening port 25555 to your server is necessary only in the case the server is on a different LAN)

**Reply by moonrise2375** [reply]
I have a Linux VM with ubuntu 18.04 running msh (with the minecraft server), It's not in the same LAN as my pc with my minecraft client. 
If I type `msh start`, the minecraft is launched and I can see the minecraft server logs.
If I launch the minecraft server without msh, I'm able to connect to the server with no problems.
Yes, I have the ubuntu firewall with the port 25555 open. I typed this command `ufw allow 25555/tcp`.

**Reply by moonrise2375** [reply]
just to make a it a bit more clear, I can open ports in the linux VM where the server and msh are hosted. What I can't do is open or forward ports in my pc where I have my minecraft client

**Reply by gekigek99** [reply]
> What I can't do is open or forward ports in my pc where I have my minecraft client

Oh ok, you don't need to do that

- Have you set the correct server address in the minecraft client? (`your.server.ip:25555`)
- Are you using minecraft server modded or vanilla?

_what I mean is that, if you are able to access the minecraft server normally at `your.server.ip:25565` then, you should be able to connect to `your.server.ip:25555` no problem (unless your host is blocking port 25555)_

**Reply by moonrise2375** [reply]
> Are you using minecraft server modded or vanilla?

I'm using fabric server with just lithium mod.

I can enter the minecraft server normally with the port 25565 but I can't with the port 25555 and I told my friends to try but they can't either. I tried adding an inbound and outbound rule to my windows firewall but that didn't work either. I'm guessing the problem is with my ISP.

**Reply by gekigek99** [reply]
Maybe you can try setting your minecraft server config on port 25555 (check if you can connect to the normal server with that port)

And then set the msh config listen port on 25565 (so that your client will connect to msh on port 25565)

**Reply by moonrise2375** [reply]
IT WORKS, THANK YOU.

Just one more question, is it normal that msh is constantly printing data/s?

**Reply by gekigek99** [reply]
Ok great! probably your server host was blocking ports different from the "standard" 25565

To deactivate that connection speed log you can set the debug option to false in the msh config
